### PR TITLE
Illustration of the effect of the different SIGHASH flag combinations

### PR DESCRIPTION
**Pull request by bilthon**
I just figured this topic could use some more visual help in order to cement the meaning of different SIGHASH flags and their combinations in the reader's mind.

If you need the SVG version of the figure in order to make further adjustments just send me a PM.

### CODE REVIEW

File: ch06.asciidoc [hunk 1]
```diff
@@ -615,6 +615,12 @@ In addition, there is a modifier flag +SIGHASH_ANYONECANPAY+, which can be combi
 | SINGLE\|ANYONECANPAY | 0x83 | Signature applies to one input and the output with the same index number
 |=======================
 
+These flag combinations are summarized in <<sighash_combinations>>
```
Comment (oboukli) [inline ch06.asciidoc]: Missing full stop?

Comment (bilthon) [reply ch06.asciidoc]: I'm sorry, but I don't think I understand your comment.

Comment (oboukli) [reply ch06.asciidoc]: There should be a `.`  (period) at the end of the sentence:
`These flag combinations are summarized in <<sighash_combinations>>.`

By the way, thank you for the illustration. It's really helpful 👍

Comment (bilthon) [reply ch06.asciidoc]: Ohh.. I see. Ok, done.

**Comment by wbnns** [comment]
@bilthon @oboukli Thank you.

**Pull request merged**